test/libs/metrics/top_level_metrics_test.py

```diff
@@ -1,40 +1,90 @@
 import dataclasses
-import io
 
 import numpy as np
 import pandas as pd
 import pytest
 import structlog
+from freezegun import freeze_time
 from covidactnow.datapublic.common_fields import CommonFields
-from covidactnow.datapublic import common_df
+
 from api import can_api_v2_definition
 from libs.metrics import top_level_metrics
 from libs.datasets.timeseries import MultiRegionDataset
 from libs.datasets.timeseries import OneRegionTimeseriesDataset
 from libs.pipeline import Region
-from freezegun import freeze_time
 
+from test.libs.datasets import timeseries_test
 from test.dataset_utils_test import read_csv_and_index_fips_date
 
+INPUT_COLUMNS = [
+    "new_cases",
+    "cases",
+    "test_positivity",
+    "positive_tests",
+    "negative_tests",
+    "contact_tracers_count",
+    "current_icu",
+    "icu_beds",
+    "current_icu_total",
+]
+
+
+def _build_timeseries_dataframe(fips, all_columns, start_date=None, dates=None, **column_data):
```

Comment:
Can you add a minimal comment about this helper?  And maybe a TODO to convert the rest of the tests to use it for their input?